pheuberger/toucan-bridge
# Implement deploy script for live networks

- Removed hardhat specific config from deploy-local.js
- Add genesis data and scripts
- Fixed forEach / for
- refactor: extract smaller functions out of `prepareToucanEnv`
- feature: enforce TCO2 to be eligible for NCT pool
- chore: remove excessive logging during test-run
- feature: add test case for non-eligible tco2
- refactor: fix typo in genesis file name
- fix: destructure object in `deploy-local.js`
- perf: update Toucan registry to immutable
- feat: allow rotating token issuer
- Tweak scripts to support multiple token owners and transfers
- Change IsNCTEligible to NotNCTEligible
- Pass tco2 contract to mintBatch
- feat: implement precision check on bridging
- test: add precision check test cases
- chore: remove unused import in test
- Config refactor
- Invalidate duplicate requests when minting TCO2s
- chore: drop redundant registry check
- Change deploy scripts to support new testing genesis method
- Upgrade contracts subrepo
- Add abacus-network/sdk
- Added allowUnlimitedContractSize to hardhat.config.js
- Adds project beneficiary to toucan.js
- Update beneficiaryAddress
- Added comment on allowUnlimitedContractSize
- Add support for Polygonscan verification
- refactor: improve variable naming in deploy function
- chore: updated submodules
- feat: add support for impersonating accounts
- feat: implement deploy script to live networks
- chore: remove cruft from hardhat config
